wgengine/router/dns_resolved.go

```diff
@@ -35,6 +36,13 @@ import (
 // this address is, in fact, hard-coded into resolved.
 var resolvedListenAddr = netaddr.IPv4(127, 0, 0, 53)
 
+// dnsReconfigTimeout is the timeout for DNS reconfiguration.
+//
+// This is useful because certain conditions can cause indefinite hangs
+// (such as improper dbus auth followed by contextless dbus.Object.Call).
+// Such operations should be wrapped in a timeout context.
+const dnsReconfigTimeout = time.Second
```

Comment:
This works mostly by coincidence, since `dns_networkmanager.go` also uses `dnsReconfigTimeout`. They are both linux-only, though, and will remain such for the foreseeable future, so it's probably fine to leave it be.

Reply:
Yeah I basically just plopped the definition where it was actually used

Reply:
Yeah, I'm noting that it is also used in another place, but it's tolerable since they always build together.